src/dev/flang/be/jvm/classfile/ClassFileConstants.java

```diff
@@ -604,6 +604,14 @@ public boolean sameAs(ClassType other)
   static ClassType JAVA_LANG_OBJECT = new ClassType("java/lang/Object");
   static ClassType JAVA_LANG_STRING = new ClassType("java/lang/String");
 
+  static ClassType NULL_TYPE = new ClassType("java/lang/Object") {
+    @Override
+    public boolean sameAs(ClassType other)
+    {
+      return true;
+    }
```

Comment:
`sameAs` should be symmetric. Better add a special case in `CodeGen.drop`: `|| jt == NULL_TYPE`.